### PR TITLE
[goecharger] fix data type for energy (nrg)

### PR DESCRIPTION
**Pull request by ranzen84**
energy (nrg) need to be double for V2
(tested with version 053.4 (Beta)

Signed-off-by: Reinhard Plaim <reinhardplaim@gmail.com>

### CODE REVIEW

File: bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerV2Handler.java [hunk 1]
```diff
@@ -152,53 +152,67 @@ protected State getValue(String channelId, GoEStatusResponseBaseDTO goeResponseB
                 if (goeResponse.sessionChargeConsumption == null) {
                     return UnDefType.UNDEF;
                 }
-                return new QuantityType<>((Double) (goeResponse.sessionChargeConsumption / 1000d), Units.KILOWATT_HOUR);
+                return new QuantityType<>((goeResponse.sessionChargeConsumption / 1000d), Units.KILOWATT_HOUR);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerV2Handler.java:155]: Can you please remove useless parenthesis. 
Same for your other changes.

**Review by lolodomo — CHANGES_REQUESTED** [review]

**Comment by lolodomo** [comment]
@SamuelBrucksch: would you like to review these changes? In particular, is it normal to remove "* 1000" or "/ 1000".

**Comment by SamuelBrucksch** [comment]
I have no idea why it is removed. Also I did not follow up the changes for V3, as I don't own that charger anymore, so I can't verify if this makes sense.

**Comment by cribskip** [comment]
I can help out with testing, however, the last build is not in jfrog anymore. How can I get a current build from this PR? Thanks :)

@lolodomo The factor 1000 is appearently not needed anymore as the values are already transmitted as needed.

**Comment by marcus-oh3** [comment]
I can also test. Will the change show up in a snapshot? If yes, are you able to guess which date?

**Review by lolodomo — APPROVED** [review]
Code looks good to me.

**Comment by lolodomo** [comment]
@ranzen84 : if you are sure of your changes, I will merge. No risk to impact handling of API V1 ?

**Pull request merged**